Model/Sales/Repository/PackageRepository.php

```diff
@@ -112,7 +113,32 @@ public function fitInDigitalStamp()
     }
 
     /**
-     * @param \Magento\Quote\Model\Quote\Item[] $products
+     * Check if the dropoff delay day has set in the products setting and get the highest delay day
+     *
+     * @param Item [] $products
+     *
+     * @return int
+     */
+    public function dropOffDelayDayWithProduct($products)
+    {
+        $highestDelayDay = 0;
```

Comment:
Misschien is ` $highestDelay` beter dan ` $highestDelayDay` 
en `$productDelay` beter dan `$productDelayDay`

Reply:
done